src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1479,16 +1551,38 @@ void ICanSaveModel.Save(ModelSaveContext ctx)
             for (int index = 0; index < memberInfos.Length; index++)
             {
                 var memberInfo = memberInfos[index];
-                var mappingAttr = memberInfo.GetCustomAttribute<LoadColumnAttribute>();
 
-                if (mappingAttr == null)
-                    throw host.Except($"{(memberInfo is FieldInfo ? "Field" : "Property")} '{memberInfo.Name}' is missing the {nameof(LoadColumnAttribute)} attribute");
+                var ignoreLoadColumnAttr = memberInfo.GetCustomAttribute<LoadColumnIgnoreAttribute>();
```

Comment:
>LoadColumnIgnoreAttribute [](start = 73, length = 25)

Why is this being added? What is the scenario where you would need to have a public field/property in the class but not have it loaded by the text loader?
If this is really necessary, then I would instead of introducing this new class, have the use the `LoadColumnNameAttribute`, and have an empty or null `Name` indicate that this field should be ignored. There is also a `NoColumnAttribute` class that can alternatively be used.